app/src/main/res/values/strings.xml

```diff
@@ -23,6 +23,7 @@
     <string name="scenario_complete">Goed gedaan!</string>
     <string name="scenario_not_complete">Het circuit klopt nog niet!</string>
     <string name="run">Uitvoeren</string>
+    <string name="scenario1_complete">Zoals je zag, is de lamp kapot gegaan. In het volgende scenario ga je leren hoe je dat kan voorkomen.</string>
```

Comment:
Iets te ingewikkeld voor kinderen?
"Helaas is het lampje nu stuk! Maar in de volgende opdracht ga je dat repareren."
Of iets dergelijks. Sowieso is het woord  'scenario' niet logisch